### DISCUSSION TITLE
참조에 의한 전달의 문제점은 무엇인가요?

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#46] [원시 값과 객체의 비교]
- 참조에 의한 전달의 문제점은 무엇인가요?

**Comment by pipisebastian** [comment]
> 참조에 의한 전달의 문제점은 무엇인가요?

여러개의 식별자가 하나의 객체를 공유한다. → 객체를 변경하면. 서로 영향을 주고받는다.

**Comment by publdaze** [comment]
> - 참조에 의한 전달의 문제점은 무엇인가요?
- 참조에 의한 전달의 경우 동일한 참조 값을 가지게 되어 참조 값에 해당하는 주소로 접근 시 동일한 데이터에 접근하여 예기치 않은 변경을 일으킬 수 있는 문제점이 있음
